packages/aws-cdk/lib/api/garbage-collection/garbage-collector.ts

```diff
@@ -12,11 +12,47 @@ import { ActiveAssetCache, BackgroundStackRefresh, refreshStacks } from './stack
 // eslint-disable-next-line @typescript-eslint/no-require-imports
 const pLimit: typeof import('p-limit') = require('p-limit');
 
-const ISOLATED_TAG = 'aws-cdk:isolated';
+const ISOLATED_TAG = 'aws-cdk.isolated';
```

Comment:
This doesn't work for ECR tags because they don't like `:` in the tag. It's a bit of a break as in it invalidates previous tags in s3, but I don't think that's a big deal at this point.